config/locales/en.yml

```
@@ -1593,7 +1593,11 @@ en:
         one: "Notification"
         other: "Notifications"
       placeholder_user: "Placeholder user"
-      project: "Project"
+      project:
+        few: "Projects"
+        many: "Projects"
+        one: "Project"
+        other: "Projects"
```

```suggestion
      project:
        one: "Project"
        other: "Projects"
```

you can omit the other keys for English. Crowdin will add appropriatey keys for different languages

---

Oh, didn't know. Thanks.